src/app/[locale]/(profile)/_components/change-password-form.tsx

```
@@ -0,0 +1,140 @@
+// src/components/dashboard/forms/ChangePasswordForm.tsx
+"use client";
+
+import { Form, FormProvider, useForm } from "react-hook-form";
```

ESLint

[error] 'Form' is defined but never used.

(@typescript-eslint/no-unused-vars)